Firestore/Example/Firestore.xcodeproj/project.pbxproj

```diff
@@ -1273,7 +1273,6 @@
 				54C9EDEE2040E16300A969CD /* Frameworks */,
 				54C9EDEF2040E16300A969CD /* Resources */,
 				EA424838F4A5DD7B337F57AB /* [CP] Embed Pods Frameworks */,
-				DE5C36328822481F6EB6EF16 /* [CP] Copy Pods Resources */,
```

Comment:
I haven't manually touched this file, so I presume all the changes come from `pod update`. Looks like gRPC renamed their dependency on OpenSSL from `openssl` to `openssl_grpc`.